packages/mui-base/src/Transitions/CssTransition.tsx

```diff
@@ -17,13 +17,6 @@ export interface CssTransitionProps {
    * is requested to exit.
    */
   exitClassName?: string;
-  /**
-   * The name of the CSS property that is transitioned the longest (has the largest `transition-duration`) on enter.
-   * This is used to determine when the transition has ended.
-   * If not specified, the transition will be considered finished end when the first property is transitioned.
-   * If all properties have the same `transition-duration` (or there is just one transitioned property), this can be omitted.
-   */
-  lastTransitionedPropertyOnEnter?: string;
```

Comment:
How were we able to remove this?  What happens in case properties have different transitions for the enter transition?